airbyte-cdk/python/unit_tests/sources/declarative/requesters/request_headers/test_interpolated_request_header_provider.py

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
I threw this test into `sources/declarative/requesters/request_headers/`, although I saw the parameter equivalent in `sources/declarative/interpolation/`. I actually kind of like it in a dedicated /request_headers folder since I see the other interpolation stuff as a base class and this (and parameters) rely on it. 

If people agree I can move `test_interpolated_request_parameter_provider.py` to its respective test folder too.

Reply:
sounds good to me!